src/documents/articles/a-quick-polymer-introduction.html.md

```diff
@@ -15,15 +15,16 @@ A guide to Polymer and its build tools.
 
 <blockquote style="margin-top:60px;margin-bottom:30px;font-style:italic;color:grey;font-size:1.5em">I've a feeling we're not in Kansas anymore - Dorothy</blockquote>
 <br/><br/>
+
 # Web component recap
 
 Web components are a set of standards for creating reusable HTML elements.
 
 For example if you wanted to create an image carousel, you could make a new html element <code>&lt;image-carousel&gt;</code> put all the JS and CSS inside, and use this element anywhere you want.
```

Comment:
`all the JS and CSS, and use this element anywhere you want` change to `all of it's internal JavaScript and CSS. After that is done, you could use this element anywhere in your document`